website/content/docs/getting-started.md

```diff
@@ -71,8 +71,20 @@ $ heron submit local \
   ~/.heron/examples/heron-examples.jar \ # The path of the topology's jar file
   com.twitter.heron.examples.ExclamationTopology \ # The topology's Java class
   ExclamationTopology # The name of the topology
+
+  INFO: Launching topology 'ExclamationTopology'
+
+  # etc
```

Comment:
By `# etc` are you indicating there's other output not being shown? In that case we typically use `...`